Fix link to point to the English docs

Miguelrom opened the pull request:
A /en was added to the link for the Performance best practices section in English.

Without it, the link redirects to the docs in some other language.

Miguelrom committed:
Fix link to point to the English docs

A /en was added to the link for the Performance best practices section
in English.

Without it, the link redirects to the docs in some other language.

jonchurch commented:
Hmm we had another PR like this recently as well, lemme see what we did. 

Do you mind experimenting with `{{page.lang}}` like in here, we may want to update everything to use this. https://github.com/expressjs/expressjs.com/pull/1441/files

Miguelrom committed:
Change anchor element for markdown link

The anchor html element was replaced by a markdown link with the page.lang variable.

Miguelrom commented:
Ok. I made the change suggested.

crandmck commented:
Thanks @Miguelrom ... Sorry for the delay.
